src/CoreShop/Component/Order/Model/PurchasableInterface.php

```diff
@@ -29,6 +29,4 @@ public function getName($language = null): ?string;
     public function getWholesaleBuyingPrice(): ?Money;
 
     public function getTaxRule(): ?TaxRuleGroupInterface;
-
-    public function getDigitalProduct(): ?bool;
```

Comment:
Symfony adds new methods via `@method bool|null getDigitalProduct()` PHPDoc on the interface before adding it for real in the next major version, so people can already see that it exists.

Reply:
@jdreesen thanks, but that method should not be there at all.